### DISCUSSION TITLE
When will --skip-install get released?

### DISCUSSION THREAD

**Discussion by vkryachko** [vercel/next.js#69750] [Help] [answered]
### Summary

This feature was implemented and merged in https://github.com/vercel/next.js/pull/64691 back in April. What is the expected timeline for it to get into https://www.npmjs.com/package/create-next-app, I can't see it even in `15.0.0-rc.0` ? thanks

**Comment by Juneezee** [comment]
It is available in the latest canary version

**Comment by vkryachko** [comment]
Cool thanks, is there an ETA for when this will be available in a stable release? thanks

**Reply by Juneezee** [reply, marked as answer]
My guess is that it will be included in the next 14.3 minor release, or the major 15 release.
